deploy/qa-int-values.yaml

```diff
@@ -0,0 +1,10 @@
+externalHost: machine-api.org
```

Comment:
We should probably change this to Serval.

Reply:
Done - in many places.